pkg/agent/store_cred.go

```diff
@@ -36,6 +28,11 @@ const (
 	terraformStateBucketNameKey string = "bucketName"
 	terraformStateAwsAccessKey  string = "awsAccessKey"
 	terraformStateAwsSecretKey  string = "awsSecretKey"
+	externalsecretns            string = "external-secrets"
+	externalsecretname          string = "nats-token"
+	natsTokenSecretPath         string = "generic/nats/auth-token"
```

Comment:
change externalsecretname to natsSecretName value "nats-token"
change externalsecretns to natsNamespace, value "observability"